pytidb/search.py

```diff
@@ -111,6 +111,7 @@ def __init__(
         self._distance_lower_bound = None
         self._distance_upper_bound = None
         self._filters = None
+        self._prefilter = False
         self._num_candidate = 20
```

Comment:
Maybe we can change it to `None`

Reply:
You mean we can use the `self._num_candidate = None` to represent the prefilter mode? Or we will rely on the code below:

```python
num_candidate = self._num_candidate if self._num_candidate else self._limit * 10
```

to make the `self._num_candidate` more flexible?

Reply:
Got it.